tools/scripts/internal-link-checker.js

```diff
@@ -0,0 +1,18 @@
+const { execSync } = require('child_process');
+const dotenv = require('dotenv');
+
+dotenv.config();
+
+const internalLinks = process.env.INTERNAL_LINKS.split(',');
```

Comment:
Please add process.exit(1) in the case when internalLinks is empty. We always need to remove internal links - this must not be empty.